Bluepill-cli/Bluepill-cli/Bluepill/Reporters/BPExecutionPhaseProtocol.h

```diff
@@ -36,6 +36,8 @@
 
 - (void)onTestCaseFailedWithName:(NSString *)testName inClass:(NSString *)testClass inFile:(NSString *)filePath onLineNumber:(NSUInteger)lineNumber wasException:(BOOL)wasException;
 
+- (void)onTestCaseAssertionFailedWithName:(NSString *)testName inClass:(NSString *)testClass;
```

Comment:
So why do we need this method? can we just take screenshot inside `onTestCaseFailedWithName`?